apps/web/pages/index.vue

```diff
@@ -100,7 +100,7 @@
       </div>
     </div>
     <NuxtLazyHydrate when-visible>
-      <NewsletterSubscribe />
+      <NewsletterSubscribe v-if="shouldDisplayComponent" />
```

Comment:
```suggestion
      <NewsletterSubscribe v-if="displayNewsletter" />
```
So that we have a parallel naming structure to the name fields toggle.